### DISCUSSION TITLE
Cannot connect Mi Home app to a brand new LX06

### DISCUSSION THREAD

**Discussion by Dinth** [duhow/xiaoai-patch#94] [Q&A] [answered]
Hiya. Im trying to connect my brand new LX06 to Mi Home app to have the speaker configured (as per the guide).
I choose the speaker in the app (the top left one, as while different from LX06, its the closest one):
![image](https://github.com/user-attachments/assets/19c9e2c8-84c6-4c41-a8bc-3fed3b0c023a)

Then im being asked to connect the phone to speakers own wifi network - no problem with connecting.

After that the speaker starts speaking chinese and the light changes, on the phone screen i am seeing this: 

![image](https://github.com/user-attachments/assets/82214959-89d7-4640-930c-ede0b51eaf4d)

But once the counter goes to around 50%, the light on the speaker changes back to orange and the counter slows down. After a while im seeing either this error:
![image](https://github.com/user-attachments/assets/68556c8e-fff7-4ec9-9db6-09983e3f69c9)

or this one:
![image](https://github.com/user-attachments/assets/001ea847-44f8-4b79-ac5c-ce83dfa65006)

What am i doing wrong?

**Comment by duhow** [comment, marked as answer]
I'm unfamiliar with [Mi Home](https://play.google.com/store/apps/details?id=com.xiaomi.smarthome) app, but the model you see is either `L05C` (China) or `L05G` (Global, Google), which are :x: unsupported.

Xiaoai Speakers from 🇨🇳 China (not `G` global models) use the [小爱音箱 mico](https://play.google.com/store/apps/details?id=com.xiaomi.mico) app.

Either case, the usage of Android apps is not supported not planned at the moment.

**Comment by Dinth** [comment]
Thanks for explaination. I was confused by this doc: 

> restart speaker and set it up using the Xiaomi Home app (wifi access -> for me, only the official app is able to set all the config files correct, otherwise you need to it over serial connection)
> choose speaker -> Speaker Pro -> enter wifi credentials -> connect to local AP from Speaker xiaomi-wifispealer-lx06...
> turn off the speaker when setting in the app is done

https://github.com/duhow/xiaoai-patch/blob/master/research/lx06/install.md
